examples/sklearnex/random_forest_classifier_spmd.py

```diff
@@ -36,8 +36,9 @@ def generate_X_y(par, seed):
     return data, resp
 
 
-params_train = {"ns": 10000, "nf": 8}
-params_test = {"ns": 100, "nf": 8}
+# TODO: Investigate out of resources error with 256 classes, accuracy issues
```

Comment:
I agree with codefactor. Please create either a GH issue or an internal ticket and remove the comment.